app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.split()[1] == command.split()[2]:
+        return None
```

Comment:
As I realized we also need to check: "Make sure you check that the `cp` input command is correct". In your case we need to check command.split()[0] == "cp". But I am not sure, so .. =)